syntax = "proto2";
package pymoliere;

message Date {
  optional int32 year = 1;
  optional int32 month = 2;
  optional int32 day = 3;
}

message Author {
  optional string first_name = 1;
  optional string last_name = 2;
}

message Journal {
  optional int32 volume = 1;
  optional int32 issue = 2;
  optional int32 year = 3;
  optional string title = 4;
}

message PubMedRecord {
  optional int32  id = 1;
  optional string raw_title = 2;
  optional string raw_abstract = 3;
  optional int32 version = 4;
  optional Date date_completed = 5;
  optional string language = 7;
  repeated Author authors = 8;
}

